controllers/loanControllers.js

```diff
@@ -1,142 +1,40 @@
 import Joi from 'joi';
```

Comment:
You still have joi here in your controller.

Reply:
let me remove it also